### PR TITLE
PROD-257/create-connect-wallet-state-on-campaigndetails

### PR DESCRIPTION
**Pull request by danijel-radomiljac**
@marvinmarnold @Yihwan I know that it says in ticket that after login it should redirect a user to deck screen. But there is some problem with nextjs navigation where I lost already a lot of time. So I would like to move on to share functionality and leave that for later.

### CODE REVIEW

**Comment by marvinmarnold** [comment]
@danijel-radomiljac conflict now

File: app/components/LoginPopUp/LoginPopUp.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        <div className="flex justify-between items-center mb-6">
+          <p className="text-base text-secondary font-bold">
+            Login to view deck
+          </p>
+          <div onClick={onClose}>
+            <CloseIcon />
+          </div>
+        </div>
+        <p className="text-sm mb-6">
+          Get started by connecting wallet to begin answer questions
```
Comment (marvinmarnold) [inline app/components/LoginPopUp/LoginPopUp.tsx:58]: The designs have typo. Should say "to begin answering questions."

**Review by marvinmarnold — CHANGES_REQUESTED** [review]

**Comment by marvinmarnold** [comment]
@Yihwan I've created two follow ups:
- https://linear.app/gator/issue/PROD-322/redirect-to-deck-after-logging-in-from-campaigns
- https://linear.app/gator/issue/PROD-323/correct-revealanswer-navigation-from-campaigns

**Review by marvinmarnold — APPROVED** [review]
Will wait for you to approve before merging @Yihwan

File: app/components/LoginPopUp/LoginPopUp.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        <div className="flex justify-between items-center mb-6">
+          <p className="text-base text-secondary font-bold">
+            Login to view deck
+          </p>
+          <div onClick={onClose}>
+            <CloseIcon />
+          </div>
+        </div>
+        <p className="text-sm mb-6">
+          Get started by connecting wallet to begin answering questions.
```
Comment (Yihwan) [inline app/components/LoginPopUp/LoginPopUp.tsx:58]:
```suggestion
          Connect your wallet to answer questions and win prizes!
```

**Review by Yihwan — APPROVED** [review]

**Pull request merged**